docs/docs/userguide/CONFIGURATION.md

```diff
@@ -64,58 +65,139 @@ region = "<REGION>"  # e.g. "ap-northeast-2"
 
     The value must be a valid [AWS region](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html){.external}.
 
+## Product specific configuration
 
```

Comment:
I think we missed database settings

Reply:
DB settings are under Optional configuration - not something I changed.

Reply:
This is the thing, we should get rid of `optional` / `mandatory` and aggregate all product related configuration.

Reply:
Fixed.